metadata-models/src/main/pegasus/com/linkedin/ml/metadata/MLFeatureSetProperties.pdl

```diff
@@ -0,0 +1,25 @@
+namespace com.linkedin.ml.metadata
+
+import com.linkedin.common.MLFeatureUrn
+import com.linkedin.common.MLPrimaryKeyUrn
+
+/**
+ * Properties associated with a MLFeatureSet
+ */
+record MLFeatureSetProperties {
+
+  /**
+   * Name of the feature set
+   */
+  name: string
```

Comment:
Should we add description here for consistency?

Reply:
Yep, I forgot to add it since it's not in Feast but I'll just include it as an optional field again